app.js

```diff
@@ -75,7 +75,7 @@ function sumArray(testArray){ //eslint-disable-line
   var addArray = sum(testArray[0],testArray[1])[0];
```

Comment:
you don't generally want to put "Array" at the end of a variable name unless it's an array. It can be a bit confusing to read.